src/test/test_crypto.c

```diff
@@ -152,8 +152,13 @@ test_crypto_openssl_version(void *arg)
   const char *h_version = crypto_openssl_get_header_version_str();
   tt_assert(version);
   tt_assert(h_version);
-  tt_assert(!strcmpstart(version, h_version)); /* "-fips" suffix, etc */
-  tt_assert(!strstr(version, "OpenSSL"));
+  if (strcmpstart(version, h_version)) { /* "-fips" suffix, etc */
+    TT_DIE(("OpenSSL library version %s did not begin with header version %s.",
+            version, h_version));
+  }
+  if (strstr(version, "OpenSSL")) {
```

Comment:
I think you want `!strstr` here.

Reply:
I don't think so -- the previous code did `assert(!strstr(...))`, but the new code does `if (strstr(...)) { DIE }`.  The `!` goes away because the sense of the test is inverted.

Reply:
Oh sorry, you're correct! Carry on.